CHANGES.rst

```diff
@@ -1,11 +1,17 @@
 Changes
 #######
+2025-08-21
+==========
+https://github.com/atviriduomenys/katalogas/pull/1846
```

Comment:
This URL points to this pull request, but it should point to the ticket w/ context instead.